runtime/lua/vim/_editor.lua

```diff
@@ -627,6 +627,113 @@ function vim.notify(msg, level, opts) -- luacheck: no unused args
   end
 end
 
+-- Helper function to decode the UTF-8 character to a Unicode code point
```

Comment:
never need to mention "helper" in function names. all functions help, else they should not exist.
```suggestion
-- Decodes a UTF-8 character to a Unicode code point.
```